### PR TITLE
My story

### PR DESCRIPTION
**Pull request by zxt5885577**
[Review on Reviewable](https://reviewable.io/reviews/udacity/create-your-own-adventure/2551)

### CODE REVIEW

File: Chinese/大话西游.md [hunk 1]
```diff
@@ -55,3 +55,5 @@
 [OnceThereIsAHill](OnceThereIsAHill/OnceThereIsAHill.md)
 
 [万里长城](world/走向世界.md)
+
+[断罪小学](crime-school/crime-school.md)
```
Comment (casey-collab) [inline Chinese/大话西游.md]: It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.

**Comment by zxt5885577** [comment]
Please review this, @casey-collab

File: Chinese/crime_school/crime_school.md [hunk 2]
```diff
@@ -0,0 +1,2 @@
+李杀神、王诛魔和刘斩仙三位少年做梦也没有想到会在狼牙区飞虎路私立断罪小学再度聚首，面对校园
```
Comment (casey-collab) [inline Chinese/crime_school/crime_school.md]: It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by zxt5885577** [comment]
Please review this, @casey-collab

**Comment by casey-collab** [comment]
Looks great! :checkered_flag:

**Pull request merged**